.github/workflows/pull-request-update.yml

```diff
@@ -1,28 +0,0 @@
-name: Pipeline
-on: pull_request
```

Comment:
should we still test to build on PRs?

Reply:
The test just moved to another workflow, we still run the test on PRs